app/packages/looker-3d/src/ErrorBoundary.tsx

```diff
@@ -15,7 +15,7 @@ const AddFo3dErrorLogs = ({
   error: Error;
   boundaryName?: string;
 }) => {
-  const thisSampleId = useRecoilValue(currentSampleId);
+  const thisSampleId = useRecoilValue(modalSampleId);
```

Comment:
In group datasets, I don't believe the `modalSampleId` necessarily represents a 3D sample. Does `looker-3d` depend on this id beyond key usage?

Reply:
This keying makes sense here for 3d status logs. We just need something "unique" about the current view. `currentSampleId` wouldn't work since it'd be null when something else is pinned.

Reply:
Ok,  just noting `@fiftyone/looker-3d` remains a singleton library until we address this

Reply:
Good point.